header/p2p/options.go

```diff
@@ -112,7 +116,9 @@ type ClientParameters struct {
 	MaxAwaitingTime time.Duration
 	// DefaultScore specifies the score for newly connected peers.
 	DefaultScore float32
-
+	// RequestDuration defines a timeout after which the session will try to re-request headers
+	// from another peer.
+	RequestDuration time.Duration
```

Comment:
Let's call it timeou, duration is too general. RequestTimeout if we want to unify it for server and client.

Reply:
Cc @renaynay, where we called it ServeTimeout

Reply:
But RequestTimeout has more sense if we unify it